### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ogsevko**
- [DEMO LINK](https://ogsevko.github.io/layout_colored-blocks/)

- [TEST REPORT LINK](https://ogsevko.github.io/layout_colored-blocks/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,8 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <h1>Colored blocks</h1>
+  <div id="b01"></div>
```
Comment (irenhh) [inline src/index.html:12]: use class names that describe the content or behavior of the data, abstract names are a bad practice

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+body {
+  margin: 0;
+}
+
+div {
```
Comment (irenhh) [inline src/style.css:5]: create a general class name for this purpose

File: src/index.html (hunk 1 above)
Comment (irenhh) [reply src/index.html:12]: ids are not recommended for styling elements, they have high specificity and they can't be reused,
use class names instead

**Review by irenhh — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  background-color: red;
+}
+
+.cgreen {
+  background-color: green;
+}
+
+.cblue {
+  background-color: blue;
+}
```
Comment (irenhh) [inline src/style.css:19]: class names are very bad, I don't understand your shortened names, the same will happen if any other developer decides to look up something in your code. You need to give specific and clear names

**Review by irenhh — CHANGES_REQUESTED** [review]

**Review by irenhh — CHANGES_REQUESTED** [review]
you didn't fix your code

**Comment by ogsevko** [comment]
I did fix it. I don't know why it's not visible in this thread though. [Here](https://github.com/mate-academy/layout_colored-blocks/commit/fa19a4d2337b64d9f61b826da7c58d226f628078 ) is my last commit.

**Review by irenhh — APPROVED** [review]
not it's fixed :)

**Pull request closed without merging**